src/testSmoke/resources/application-smoke-test.yml

```diff
@@ -11,14 +11,14 @@ spring:
             scope: read
         provider:
           hmpps-smoketest-auth:
-            token-uri: ${smoketest.hmpps-auth.endpoint.url}/oauth/token
+            token-uri: ${smoketest.oauth.endpoint.url}/oauth/token
 
 graceful:
   shutdown.enabled: false
 
 smoketest:
   enabled: true
-  hmpps-auth:
```

Comment:
Turns out this is overridden in some Circle env vars for context `hmpps-smoketest-common-vars` so we can't change here without changing the Circle env vars.